spring-boot/README.md

```diff
@@ -95,7 +100,7 @@ the application to Java SE on App Service:
 * Use Maven to deploy the application from the `spring-boot` directory:
 
 ```
-mvn clean package azure-webapp:deploy
+mvn clean package -Pazure azure-webapp:deploy
```

Comment:
Is a separate Maven profile absolutely necessary? Can the Azure DB starter work with a local database with passwords?

Reply:
The Azure DB starter supports local database. While we don't need the Azure DB starter locally, so the `azure` profile was introduced. I tried to include the dependency in webapp maven plugin, but the application didn't work.
It's good to include Azure DB starter in local test. I will remove the profile.